packages/core/src/baseclient.ts

```diff
@@ -58,6 +58,9 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
   /** Is the client still processing a call? */
   protected _processing: boolean = false;
 
+  /** Processing interval */
+  protected _processingInterval?: NodeJS.Timeout;
```

Comment:
I'm not sure if this will be available in browser environments, which may cause same issus as with `getCurrentHub` and `Window`

Reply:
Good catch!